src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/ClaimExpenseAPITestCases.yaml

```diff
@@ -0,0 +1,212 @@
+Create:
```

Comment:
better if can test for different scenarios 

- wrong claimRequestId, expenseTypeId
- amount, note, date with wrong or null values